package.json

```diff
@@ -16,6 +17,7 @@
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "axios": "^0.27.2",
+    "dotenv": "^16.0.1",
```

Comment:
why this was added?

Reply:
dealt with different ways of working with env.
already removed this package